### PR TITLE
Image refresh for rhel-8-1

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for rhel-8-1
 * [x] image-refresh rhel-8-1
 * [x] Add workaround to rhel-8.1 branch: PR #12860
 * [x] Add workaround to rhel-8-appstream branch: PR #12861

### CODE REVIEW

**Comment by martinpitt** [comment]
Pretty well all tests fail on this:
```
audit: type=1400 audit(1569048718.494:5): avc:  denied  { dac_override } for  pid=1700 comm="rhsmd" capability=1  scontext=system_u:system_r:rhsmcertd_t:s0-s0:c0.c1023 tcontext=system_u:system_r:rhsmcertd_t:s0-s0:c0.c1023 tclass=capability permissive=0
```
This can't become a naughty, we need a global override in testlib.py.

**Comment by martinpitt** [comment]
This was already reported at https://bugzilla.redhat.com/show_bug.cgi?id=1753991 , I followed up with some details.

**Comment by martinpitt** [comment]
I pushed the testlib workaround. This also needs to be added to the rhel-8.1 branch.

**Comment by martinpitt** [comment]
I backported the workaround to the rhel-8.1 branch in #12860. That needs to land first.

File: test/common/testlib.py [hunk 1]
```diff
@@ -932,6 +932,10 @@ def check_journal_messages(self, machine=None):
             # Fedora >= 30 switched to dbus-broker
             self.allowed_messages.append("dbus-daemon didn't send us a dbus address; not installed?.*")
 
+        if self.image in ['rhel-8-1', 'rhel-8-1-distropkg']:
```
Comment (marusak) [inline test/common/testlib.py]: Any reason you did it like this and not with naughty? Is problem that distropkg is affected?
Known issues are easier to follow and less likely we just forget about them. Also with naughty we would not need to backport it.

Comment (martinpitt) [reply test/common/testlib.py]: This bug affects most/all tests, and naughties just don't scale well. They are meant for ignoring one or two failed tests, not hundreds. Updating the known issue via the github API tens of thousands of times is just not practical.

**Comment by martinpitt** [comment]
@marusak : This works now, can you please review/land?

**Pull request merged**